### DISCUSSION TITLE
Segmentation Fault

### DISCUSSION THREAD

**Discussion by Vabu77** [IIC2133-PUC/2024-1#59] [Tarea 0]
Hola,

No logro encontrar el error que hace que ocurra un segmentation fault en mi código al probar el test parte-3-hard. No entiendo porque ocurre esto ya que al correr los otros test tanto de la misma parte 3 como de la 2 y 1 no ocurre esto. Gracias!

**Comment by benjavicente** [comment]
Hola. Lo último que tienes subido tiene errores de sintaxis, así que no se puede probar ver ese error.

**Reply by Vabu77** [reply]
Ahora subí una versión nueva. Igual yo no he tenido problemas para correrlo, puede ser que me falte subir algo?
